.gitignore

```diff
@@ -51,4 +51,4 @@ node_modules/
 
 # Build output
 gatsby-node.js
-normalize.js
+utils.js
```

Comment:
@ajayns I guessed that the filename "normalize.js" came from other plugin and renamed it into more generic "utils.js"

Reply:
I was unsure what would be the best name, so I just looked through other plugins. This makes more sense now, agreed.